hw/ip/power_manager/data/power_manager.sv.tpl

```diff
@@ -481,8 +485,8 @@ module power_manager #(
       .rst_ni,
 
       // trigger to start the sequence
-      .start_off_sequence_i(reg2hw.power_gate_ram_block_${bank}.q),
-      .start_on_sequence_i (~reg2hw.power_gate_ram_block_${bank}.q),
+      .start_off_sequence_i(reg2hw.power_gate_ram_block_${bank}.q || reg2hw.set_retentive_ram_block_${bank}.q),
```

Comment:
are you sure you want to use the same fsm ? why you put this condition in OR?

Reply:
I would like the iso cells are activated even when the block is in retentive and not power gated. Just for safety.